README.md

```diff
@@ -0,0 +1,179 @@
+only 95% of the packets will be passed in each of the subnet, i.e. 5% of the data will be lost through the subnet
+
+**Receiver** - The behavior of receiver is to receive the data and send back an acknowledgement extracted from the received
+data after a time period.
+
+---
+
+## File Structure
+---
+
+##### bin [This folder containes the .exe files which are generated during program execution]
+1. src [This folder contains the simulator .exe file]
+	- ABP.exe
+2. test [This folder contains the test .exe files]
+	- SUBNET_TEST.exe
+	- SENDER_TEST.exe
+	- RECEIVER_TEST.exe
+		
+##### build [This folder contains .o files which are generated during program execution]
+1. src [This folder contains simulator .o files]
+	- main.o
+	- message.o
+2. test [This folder contains the test .o files]
+	1. subnet
+		- main.o
+	2. sender
+		- main.o
+	3. receiver
+		- main.o
+			
+##### data [This folder contains input data for testing and output files]
+1. input_abp_0.txt
+2. input_abp_1.txt
+3. abp_output.txt
+	
+##### doc [This folder contains documents related to project]
+1. alternatebitprot.pdf
+2. Cadmium_Documentation_Ubuntu.pdf
+3. Cadmium_Windows.pdf
+		
+##### include [THis folder contains the header files and data structures]
+1. atomics [This folder contains the header files]
+	- receiverCadmium.hpp
+	- senderCadmium.hpp
+	- subnetCadmium.hpp
+2. data_structures [This folder contains the data structures used in the project]
+	- message.hpp
+	- message.cpp
+
+##### lib [This folder contains the libraries/dependencies]
+1. cadmium [This folder contains cadmium files]
+2. DESTimes [This folder contains DESTimes files]
+3. vendor [This folder contains 3rd party header files needed in the project]
+	- iestream.hpp
+	- NDTime.hpp
+		
+##### src [This folder contains source code for the project]
+1. data_structures [This folder contains the data structures used in the project]
+	- message.cpp
+2. top_model [This folder contains source code for the Alternate Bit Protocol simulator]	
+	- main.cpp
+
+##### test [This folder contains the unit test for the different include files]
+1. data
+	- receiver_input_test.txt
+	- receiver_test_output.txt
+	- sender_input_test_ack_In.txt
+	- sender_input_test_control_In.txt
+	- sender_test_output.txt
+	- subnet_input_test.txt
+	- subnet_test_output.txt
+2. src
+	1. receiver [This folder contains the unit test of the receiver]
+		- main.cpp
+	2. sender [This folder contains the unit test of the sender]
+		- main.cpp
+	3. subnet [This folder contains the unit test of the subnet]
+		- main.cpp
+
+--- 
+## Installation Guide 
+---
+
+Following the corresponding link [Installation Guide document](https://github.com/abhinavachanta/AlternateBitProtocolSimulator/tree/master/doc)instructions to install cadmium simulator dependencies on windows operating system. Follow the corresponding link [https://github.com/abhinavachanta/AlternateBitProtocolSimulator/blob/master/doc/Cadmium_Windows.pdf] guides to install cadmium on windows.
```

Comment:
1) give space after (https://github.com/abhinavachanta/AlternateBitProtocolSimulator/tree/master/doc) and between instructions.
2) not a good way of creating a link.
link [https://github.com/abhinavachanta/AlternateBitProtocolSimulator/blob/master/doc/Cadmium_Windows.pdf]

Reply:
it's good now